AreaOfTriangle.js

```diff
@@ -1,6 +1,6 @@
 function Area(base, height) {
-    AreaofT = 0.5 * base * height;
-    return AreaofT;
+    return 0.5 * base * height;
+
 
 }
 AreaOfATriangle = Area(3, 4);
```

Comment:
- functions should start with a small letter.
- your variable AreaOfATriangle should also start with a small letter

Reply:
Done